src/libstd/time/duration.rs

```diff
@@ -83,7 +83,7 @@ impl Duration {
 
     /// Returns the number of whole seconds represented by this duration.
     ///
-    /// The extra precision represented by this duration is ignored (e.g. extra
+    /// The extra precision represented by this duration is ignored (i.e. extra
```

Comment:
I am not 100% sure of this one, mostly because I don't remember the exact semantics of this function.